CHANGELOG.md

```diff
@@ -2,6 +2,34 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 17 end 2015-01-05
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+17+Ending+Jan+5%2C+2015%22+label%3Aenhancement)
+
+- Rest API: Added VM Management Actions
```

Comment:
correct, to be merged in sprint 18 so we can remove from the Sprint 17 README.md.